well_profile/generator.py

```diff
@@ -14,11 +14,11 @@ def two_points(points):
         a wellpath object with 3D position
     """
 
-    if 'north' not in points['kickoff']:
-        points['kickoff']['north'] = 0
+    #if 'north' not in points['kickoff']:
```

Comment:
I'm not sure why this under comment 🤔. This and the couple of lines below (17, 18, 20, 21) just set kickoff location at (0,0) if it's not provided.